packages/calypso-analytics/tsconfig.json

```
@@ -12,7 +12,7 @@
 		"allowJs": false,
 		"jsx": "react",
 		"declaration": true,
-		"declarationDir": "dist/types",
+		"declarationDir": "types",
```

Please use `dist/types` like the rest of the packages:

```suggestion
		"declarationDir": "dist/types",
```